alvr/system_info/src/lib.rs

```diff
@@ -120,9 +126,11 @@ pub fn platform() -> Platform {
             ("Oculus", _, "seacliff") => Platform::QuestPro,
             ("Oculus", _, _) => Platform::QuestUnknown,
             ("Pico", "Pico Neo 3" | "Pico Neo3 Link", _) => Platform::PicoNeo3,
-            ("Pico", _, "PICOA8110" | "phoenix") => Platform::Pico4,
-            ("Pico", _, "sparrow") => Platform::Pico4Ultra,
-            ("Pico", _, "merline") => Platform::PicoG3,
+            ("Pico", "A8A10", _) => Platform::Pico4Pro,
+            ("Pico", "A8E50", _) => Platform::Pico4Enterprise,
+            ("Pico", "A8110", _) => Platform::Pico4,
+            ("Pico", "A9210", _) => Platform::Pico4Ultra,
+            ("Pico", "A7Q10", _) => Platform::PicoG3,
```

Comment:
What is the source for these codes? the reason why I did not split Pico4Pro and Pico4Enterprise is because @Meister1593 reported that their Pico 4 pro eye has DEVICE code A8110 (which corresponds to Pico4 in the mapping above), so I thought the codes were not reliable to detect the device type.

Reply:
Logcat shows `manufacturer: Pico, model: A8A10, device: PICOA8110` for me.

I've asked some people before about model info on their Pico 4 Enterprise and Pico 4 Ultra. Model names were also found in settings application and user manuals.

Reply:
I see, so you crowdsourced the info. and yeah that was MANUFACTURER, not DEVICE. I'm ok for merging then